src/hooks/useRedeem.ts

```diff
@@ -6,6 +6,10 @@ import { useTransactions } from "./useTransactions";
 import { ethers } from "ethers";
 import { getKpiOptionsEMP, getKpiTokenContract } from "../utils";
 
+// max uint value is 2^256 - 1
+const MAX_UINT_VAL = ethers.BigNumber.from(2).pow(256).sub(1);
```

Comment:
[`ethers.constants.MaxUint256 ⇒ BigNumber`](https://docs.ethers.io/v5/single-page/#/v5/api/utils/constants/)

Reply:
thank you, changed!